UE/Application/States/ConnectedState.cpp

```diff
@@ -36,22 +36,22 @@ void ConnectedState::handleShowSms(IUeGui::IListViewMode::Selection indexOfSms)
     context.user.viewSms(retrievedSms);
 }
 
-void ConnectedState::handleCallRequest(common::PhoneNumber from)
+void ConnectedState::handleReceiveCallRequest(common::PhoneNumber from)
 {
     using namespace std::chrono_literals;
     context.user.showCallRequest(from);
     context.timer.startTimer(30s);
 }
 
-void ConnectedState::handleCallAccept(common::PhoneNumber to)
+void ConnectedState::handleSendCallAccept(common::PhoneNumber to)
 {
     context.bts.sendCallAccepted(to);
     context.user.showTalking();
     context.timer.stopTimer();
     context.setState<TalkingState>();
 }
 
-void ConnectedState::handleCallDrop(common::PhoneNumber to)
+void ConnectedState::handleSendCallDrop(common::PhoneNumber to)
```

Comment:
forgot to override "handleStartDial", "handleSendCallRequest" and "handleReceiveCallAccept".